tests/zoltan/zoltan_01.cc

```diff
@@ -0,0 +1,90 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2004 - 2015 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+
+// check GridTools::partition_triangulation
```

Comment:
you probably use some other test as an example, it's good to state it here so that in case something goes wrong 
in years to come, it's easier to debug.

You can also be a bit more specific by saying

> check GridTools::partition_triangulation using Zoltan